terminus-ui/src/radio-group/radio-group.component.ts

```diff
@@ -277,7 +277,14 @@ export class TsRadioGroupComponent extends TsReactiveFormBaseComponent implement
    * Define if the visual style should be large or small
    */
   @Input()
-  public small: boolean = false;
+  public set small(value: boolean) {
+    this._small = coerceBooleanProperty(value);
+    this.value = this._small;
```

Comment:
We don't want to set the value here. `this.value` is the underlying model for any form components where we implement our own `ControlValueAccessor`.

Reply:
And to the point you made earlier today about adding integration tests here next... they would have caught this 😺